chef_master/source/upgrade_server.rst

```diff
@@ -14,6 +14,10 @@ There are two upgrade scenarios for upgrades from earlier versions of Chef serve
 * `Standalone </upgrade_server.html#standalone>`__
 * `High availability </upgrade_server.html#high-availability>`__
 
+.. note:: As of version 12.14, Chef server will not render passwords outside of the ``/etc/opscode`` directory by default. If you are not using any Chef server add-ons, or you're using the latest add-on versions, you can set ``insecure_addon_compat`` to ``false`` in ``/etc/opscode/chef-server.rb``. With this option set to ``false``, Chef server writes all credentials to a single location.
```

Comment:
Maybe a note here that it's important to *not* set the flag before Chef Server then the add-ons are upgraded ? The wrong order will break the add-ons, requiring manual intervention to fix them.